doc/tutorial/basic/tutorial.rst

```diff
@@ -62,11 +62,11 @@ We can separate learning problems in a few large categories:
 .. topic:: Training set and testing set
 
     Machine learning is about learning some properties of a data set
-    and applying them to new data. This is why a common practice in
-    machine learning to evaluate an algorithm is to split the data
-    at hand into two sets, one that we call the **training set** on which
-    we learn data properties and one that we call the **testing set**
-    on which we test these properties.
+    and then testing those properties against another data set. A common practice in
```

Comment:
We try to keep lines under 80 chars

Reply:
Thanks. I was concerned with reflowing text, but I'll make the changes.